src/types/form.ts

```diff
@@ -197,27 +197,33 @@ export type UseFormRegister<TFieldValues extends FieldValues> = <
   options?: RegisterOptions<TFieldValues, TFieldName>,
 ) => UseFormRegisterReturn;
 
+export type SetFocusOptions = Partial<{
+  shouldSelect: boolean;
+}>;
+
 /**
  * Set focus on a registered field. You can start to invoke this method after all fields are mounted to the DOM.
  *
  * @remarks
  * [API](https://react-hook-form.com/api/useform/setfocus) • [Demo](https://codesandbox.io/s/setfocus-rolus)
  *
  * @param name - the path name to the form field value.
+ * @param options - input focus behavior options
  *
  * @example
  * ```tsx
  * useEffect(() => {
  *   setFocus("name");
  * }, [setFocus])
- *
- * <button onClick={() => setFocus("name")}>Focus</button>
+ * // shouldSelect allows to select input's content on focus
+ * <button onClick={() => setFocus("name", { shouldSelect })}>Focus</button>
```

Comment:
```suggestion
 * <button onClick={() => setFocus("name", { shouldSelect: true })}>Focus</button>
```

I think set to true is missing here.

Reply:
yep, updated, thx